sklearn/compose/tests/test_column_transformer.py

```diff
@@ -431,11 +431,13 @@ def test_make_column_transformer_kwargs():
     scaler = StandardScaler()
     norm = Normalizer()
     ct = make_column_transformer(('first', scaler), (['second'], norm),
-                                 n_jobs=3, remainder='drop')
+                                 n_jobs=3, remainder='drop',
+                                 sparse_threshold=0.3)
```

Comment:
Can you use a different default here? Otherwise the test below is not checking that the correct value is actually passed through (as 0.3 is the default)

Reply:
I just pushed it -again - too late?

Reply:
yeah, don't worry (we are in a hurry a bit to get a release done).
Already opened a follow-up PR: https://github.com/scikit-learn/scikit-learn/pull/12156

Reply:
But thanks for the PR!